dev-haskell/apply-refact/apply-refact-0.14.0.0.ebuild

```
@@ -0,0 +1,40 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# ebuild generated by hackport 0.8.5.0.9999
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Perform refactorings specified by the refact library"
+HOMEPAGE="https://github.com/mpickering/apply-refact"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~amd64"
+
+RDEPEND=">=dev-haskell/extra-1.7.3[profile?]
+	>=dev-haskell/filemanip-0.3.6.3[profile?] <dev-haskell/filemanip-0.4[profile?]
+	dev-haskell/ghc-exactprint[profile?]
+	dev-haskell/ghc-paths[profile?]
+	>=dev-haskell/optparse-applicative-0.15.1.0[profile?]
+	>=dev-haskell/refact-0.2[profile?]
+	>=dev-haskell/syb-0.7.1[profile?]
+	>=dev-haskell/uniplate-1.6.13[profile?]
+	>=dev-haskell/unix-compat-0.5.2[profile?]
+	>=dev-lang/ghc-9.2[profile?] <dev-lang/ghc-9.3[profile?]
+	>=dev-lang/ghc-9.2.4
```

I would just remove the extra `ghc` dependencies.

```suggestion
	>=dev-lang/ghc-9.2.4:=
```